site/src/paradox/dev/Style-Guide.md

```diff
@@ -4,52 +4,64 @@
+#### `Sbt` plugin 
 
-### IntelliJ IDEA
+Run the following command to format the entire codebase.
 
-Most of us write Scala code in IntelliJ IDEA and it's wise to let the IDE do most of the work including managing imports and formatting code. We also want to avoid custom settings as much as possible to make on-boarding new developers easier. Hence we use IntelliJ IDEA's default settings with the following exceptions:
+`sbt scalafmt test:scalafmt scalafmtSbt`
 
-- Set _Code Style &rarr; Default Options &rarr; Right margin (columns)_ to 100
-- Turn off _Code Style &rarr; Scala &rarr; ScalaDoc &rarr; Use scaladoc indent for leading asterisk_
-- Under _Copyright &rarr; Copyright Profiles_, add the following template.
+#### IntelliJ IDEA
 
-```
-Copyright $today.year Spotify AB.
+Most of us write Scala code in IntelliJ IDEA and it's wise to let the IDE do most of the work including managing imports and formatting code. To use `scalafmt` you need an additional plugin. Follow this [link](https://plugins.jetbrains.com/plugin/8236-scalafmt) to install it.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+We also want to avoid custom settings as much as possible to make on-boarding new developers easier. Hence we use IntelliJ IDEA's default settings with the following exceptions:
 
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing,
-software distributed under the License is distributed on an
-"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-KIND, either express or implied.  See the License for the
-specific language governing permissions and limitations
-under the License.
-```
+- Set _Code Style &rarr; Scala &rarr; Formatter &rarr; scalafmt_
+- Under _Copyright &rarr; Copyright Profiles_, add the following template.
+  
+  ```
+  Copyright $today.year Spotify AB.
+  
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+  
+      http://www.apache.org/licenses/LICENSE-2.0
+  
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+  ```
 
 ### ScalaStyle
 
-We use ScalaStyle to cover the basics and the entire code base should pass. In case of exceptions, wrap the violating code with a pair of comments to temporarily suppress the warning.
+We use ScalaStyle to cover other rules that we don't cover with `scalafmt` and the entire code base should pass. In case of exceptions, wrap the violating code with a pair of comments to temporarily suppress the warning.
```

Comment:
it might be helpful to link to the scalastyle docs that explain this comment syntax